backend/src/graphql/resolver/UserResolver.test.ts

```diff
@@ -1578,39 +1693,83 @@ describe('UserResolver', () => {
         describe('change role with success', () => {
           beforeAll(async () => {
             user = await userFactory(testEnv, bibiBloxberg)
+            admin = await userFactory(testEnv, peterLustig)
+            await mutate({
+              mutation: login,
+              variables: { email: 'peter@lustig.de', password: 'Aa12345_' },
+            })
+          })
+
+          afterAll(async () => {
+            await cleanDB()
+            resetToken()
           })
 
           describe('user gets new role', () => {
             describe('to admin', () => {
+              it('returns admin-rolename', async () => {
+                const result = await mutate({
+                  mutation: setUserRole,
+                  variables: { userId: user.id, role: ROLE_NAMES.ROLE_NAME_ADMIN },
+                })
+                expect(result).toEqual(
+                  expect.objectContaining({
+                    data: {
+                      setUserRole: ROLE_NAMES.ROLE_NAME_ADMIN,
+                    },
+                  }),
+                )
+              })
+
+              it('stores the ADMIN_USER_ROLE_SET event in the database', async () => {
+                const userContact = await UserContact.findOneOrFail({
+                  where: { email: 'bibi@bloxberg.de' },
+                  relations: ['user'],
+                })
+                const adminContact = await UserContact.findOneOrFail({
+                  where: { email: 'peter@lustig.de' },
+                  relations: ['user'],
+                })
```

Comment:
Why this?

Reply:
testing if the events where written still exists. it seems to be a diff confusion

Reply:
outdated